__tests__/__fixtures__/fixtures.ts

```diff
@@ -22,4 +26,12 @@ export default {
     const eslintWithTodos = deepCopy(fixtures.eslintWithTodos);
     return tmp ? updatePaths(tmp, eslintWithTodos) : eslintWithTodos;
   },
+  eslintWithErrorsWarningsTodos(tmp?: string): ESLint.LintResult[] {
+    const eslintWithErrorsWarningsTodos = deepCopy(
+      fixtures.eslintWithErrorsWarningsTodos
+    );
+    return tmp
```

Comment:
Under what circumstance to we not want to provide a tmp dir to update the paths in the fixture with?

Reply:
True, I removed it and made the default return.